wpilibc/src/main/native/cpp/InterruptableSensorBase.cpp

```diff
@@ -17,25 +17,28 @@ using namespace frc;
 InterruptableSensorBase::~InterruptableSensorBase() {
   if (m_interrupt == HAL_kInvalidHandle) return;
   int32_t status = 0;
-  auto param = HAL_CleanInterrupts(m_interrupt, &status);
-  if (param) {
-    delete reinterpret_cast<InterruptEventHandler*>(param);
-  }
+  HAL_CleanInterrupts(m_interrupt, &status);
   // Ignore status, as an invalid handle just needs to be ignored.
   m_interrupt = HAL_kInvalidHandle;
+  m_interruptHandler = nullptr;
```

Comment:
What's the point of assigning to member variables in the destructor?

Reply:
Good point, I'll remove those.